docs/src/main/sphinx/security/authentication-types.rst

```diff
@@ -0,0 +1,62 @@
+====================
+Authentication types
+====================
+
+Trino supports multiple authentication types to ensure all users of the system
+are authenticated. Different authenticators allow user management in one or more
+systems. Using :doc:`TLS <tls>` is required for all authentications types.
+
+You can configure one or more authentication types with the
+``http-server.authentication.type`` property. The following authentication types
+and authenticators are available:
+
+* ``PASSSWORD`` for  :doc:`password-file`, :doc:`ldap`, and :doc:`salesforce`
+* ``OAUTH2`` for :doc:`oauth2`
+* ``CERTIFICATE`` for certificate authentication
+* ``JWT`` for Java Web Token (JWT) authentication
+* ``KERBEROS`` for :doc:`Kerberos authentication <server>`
+
+The simplest configuration to get started is password authentication backed by a
+password file:
+
+.. code-block:: properties
+
+    http-server.authentication.type=PASSWORD
+
+
+Multiple authentication types
+-----------------------------
+
+You can use multiple authentication types, separating with commas in the
+configuration:
+
+.. code-block:: properties
+
+    http-server.authentication.type=PASSWORD,CERTIFICATE
+
+Authentication is performed in order of the entries, and first successful
+authentication results in access.
+
+Multiple password authenticators
+--------------------------------
+
+You can use multiple password authenticators types by referencing multiple
+configuration files:
+
+.. code-block:: properties
+
+    http-server.authentication.type=PASSWORD,CERTIFICATE
+    password-authenticator.config-files=etc/ldap1.properties,etc/ldap2.properties,etc/password.properties
+
+In the preceding example, the configuration files ``ldap1.properties`` and
+``ldap1.properties`` are regular :doc:`LDAP authenticator configuration files
```

Comment:
I'd recommend "previous" instead of "preceding"

Reply:
Google style is preceding..